bundle/manifests/oadp-operator.clusterserviceversion.yaml

```diff
@@ -1190,6 +1202,40 @@ spec:
                       path: token
               - emptyDir: {}
                 name: tmp-dir
+      - label:
+          app: oadp-cli
+        name: openshift-adp-oadp-cli-server
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              app: oadp-cli
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                app: oadp-cli
+            spec:
+              containers:
+              - image: quay.io/rh_ee_jvaikath/oadp-cli-binaries
```

Comment:
I think it would be neat if this container is self contained and can also create the CLIDownload CR. but if it end up being easier to do via oadp-operator manager container then that's fine too.